src/views/Swap/Swap.vue

```diff
@@ -502,55 +500,102 @@ export default {
       signRequestModalOpen: false,
       swapErrorMessage: '',
       customFeeAssetSelected: null,
-      customFees: {},
-      bridgeModalOpen: false
+      customFees: {}
     }
   },
   props: {
     routeAsset: String,
     accountId: String
   },
-  created() {
+  async created() {
     this.asset = this.routeAsset
     this.fromAccountId = this.accountId
-    this.fee = this.fees[this.selectedFee || 'average']?.fee
-    this.updateMarketData({ network: this.activeNetwork })
-    ;(async () => {
-      await this.updateFees({ asset: this.assetChain })
-      await this.updateMaxSwapFees()
-    })()
-
-    let toAsset = null
+    let _sendAmount = 0.0
+    let _toAsset = null
+
+    if (this.$route.query.mode === 'tab') {
+      const {
+        toAccountId,
+        sendAmount,
+        toAsset,
+        selectedFees,
+        customFees,
+        currentStep,
+        maxOptionActive,
+        provider,
+        userSelectedQuote
+      } = this.$route.query
+      this.currentStep = currentStep
+      this.maxOptionActive = maxOptionActive
+      this.toAccountId = toAccountId
+      _toAsset = toAsset
+      _sendAmount = sendAmount
+
+      this.selectedFee = selectedFees.split(',').reduce((prev, curr) => {
```

Comment:
instead of splitting `,` and `=`
it might be better to parse it first using `qs` library, then do the rest

Reply:
fixed